src/main/java/network/brightspots/rcv/Tabulator.java

```diff
@@ -59,20 +60,20 @@ class Tabulator {
   // roundTallies is a map from round number to a map from candidate ID to vote total for the round
   // e.g. roundTallies[1] contains a map of all candidate ID -> votes for each candidate in round 1
   // this structure is computed over the course of tabulation
-  private final Map<Integer, RoundTally> roundTallies = new HashMap<>();
-  // precinctRoundTallies is a map from precinct to roundTallies for that precinct
-  private final Map<String, Map<Integer, RoundTally>> precinctRoundTallies = new HashMap<>();
+  private final RoundTallies roundTallies = new RoundTallies();
+  // roundTalliesByField is a map from a tabulate-by field to roundTallies for that field
+  private final BreakdownByField<RoundTallies> roundTalliesByField = new BreakdownByField<>();
```

Comment:
Am I misreading the code, or does this mean that if you're using more than one breakdown (with the current code, that would be both batch and precinct), there will be a collision if there's any overlap in the IDs? E.g. if there's a batch 1 and also a precinct 1, they'll be writing to and reading from the same tallies?

Reply:
Ah, no, now I see that `roundTalliesByField` is a map from field to fieldId to round to vote count. Maybe `roundTalliesByFields` (plural) would be more clear?